.github/workflows/parse_changelog_workflow.yml

```diff
@@ -4,6 +4,10 @@ on:
   pull_request:
     types: [opened, edited]
 
+permissions:
+  pull-requests: read
+  issues: write
```

Comment:
Pull requests cannot get `write` permissions since anyone can trigger them, if you need write permissions for this workflow consider look into using `pull_request_target` instead.

For more context see https://stackoverflow.com/a/74959635/533057

Reply:
Hi @peternied,

Thanks for the information. However, that commit in particular is outdated. I already did your suggestion before on commit [84441e](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/5519/commits/84441e3fb8169715b3fe0843fb9927ec244b77b5)